app/init.php

```diff
@@ -89,7 +89,7 @@
 const DELETE_TYPE_AUDIT = 'audit';
 const DELETE_TYPE_ABUSE = 'abuse';
 const DELETE_TYPE_CERTIFICATES = 'certificates';
-const DELETE_TYPE_USAGE_STATS = 'certificates';
+const DELETE_TYPE_USAGE_STATS = 'usageStats';
```

Comment:
Let keep the name to a single word when possible.
```suggestion
const DELETE_TYPE_USAGE = 'usage';
```